sdk/search/Microsoft.Azure.Search/tests/Tests/SynonymMapTests.cs

```diff
@@ -265,6 +283,7 @@ public void ExistsReturnsTrueForExistingSynonymMap()
         }
 
         [Fact]
+		[LiveTest]
```

Comment:
nit: indention

Reply:
@weshaggard Why is this new attribute required?

Reply:
For our pipelines we filter tests based on category of "Live" and only run those nightly and not in CI. So my assumption is that these were live tests, correct @chidozieononiwu?

Reply:
All our tests use the Azure test framework and are capable of running in Playback mode. They should *all* run in CI, and should complete in a few seconds at most.

Reply:
Ok I just chatted with @chidozieononiwu and I believe he was hitting those issues you called out in your email and thought it was because they were trying to run against the live service. He is going to undo the addition of these attributes and ensure all the tests pass for CI.

Reply:
I have gotten rid of the LiveTest Attibutes. I had to add the DisableTestRunParallel.cs import for dataplane tests to get it to build.